<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Regression Test Suite"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="35.635">
  <properties />
            <testcase name="testAmazonHomePage" time="10.876" classname="tests.AmazonWebProductSearch">
                <failure type="java.lang.AssertionError"
                  message="expected [19] but found [21]"
              >
<![CDATA[java.lang.AssertionError: expected [19] but found [21]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertEqualsImpl(Assert.java:137)
  org.testng.Assert.assertEquals(Assert.java:118)
  org.testng.Assert.assertEquals(Assert.java:652)
  org.testng.Assert.assertEquals(Assert.java:662)
  tests.AmazonWebProductSearch.testAmazonHomePage(AmazonWebProductSearch.java:55)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:571)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
  org.testng.TestNG.runSuites(TestNG.java:1028)
  org.testng.TestNG.run(TestNG.java:996)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="testAmazonHomePage" time="13.275" classname="tests.AmazonWebProductSearch" />
                <testcase name="testAmazonHomePage" time="11.484" classname="tests.AmazonWebProductSearch" />
      
</testsuite>
